helm-charts/yunikorn/templates/service.yaml

```diff
@@ -30,10 +30,12 @@ spec:
       targetPort: http1
       protocol: TCP
       name: yunikorn-core
+    {{- if .Values.enableWebService }}
```

Comment:
Could you please apply it to ingress also? If there is no web port, we don't need to configure ingress, right?

Reply:
Okay! I have applied the changes to the ingress.